.github/workflows/examples.yml

```diff
@@ -60,13 +64,8 @@ jobs:
     - name: Seismic elastic examples
       shell: bash -l {0}
       run: |
-        py.test examples/seismic/elastic/elastic_example.py
-        py.test examples/seismic/viscoelastic/viscoelastic_example.py
-
-    - name: Seismic viscoacoustic examples
-      shell: bash -l {0}
-      run: |
-        py.test examples/seismic/viscoacoustic/viscoacoustic_example.py
```

Comment:
Dropping because... ?

Reply:
All tests are run above......

Reply:
right, I guess I'm an idiot again. So the py.test command searches for tests recursively in all subdirectories in all files named `*_example.py example_*.py`?

Reply:
yep